### ISSUE TITLE
Name

### ISSUE THREAD

**Issue by skddc** [67P/meta#1]
As nobody proposed new names on https://wiki.67p.io/index.php/Naming#Candidates and nobody had major objections against my current proposal, let's fix this and move on.

Everybody in favor, please add your +1 to this issue or comment with objections!

5apps will front the money for the domain and expect to be remunerated later, no matter in which way. Our basic agreement about rewarding contributions should apply, so nobody ows someone else money there.

/cc @bkero @bumi @dkgrieshammer @galfert @gregkare @janlelis @silverbucket

**Comment by gregkare** [comment]
+1

**Comment by galfert** [comment]
+1 for Kosmos

**Comment by janlelis** [comment]
:+1: Kosmos

**Comment by silverbucket** [comment]
So 5apps would purchase `kosmos.org` for ~2795 USD? That is a lot of money to put down for this name. I think the name Kosmos is OK, but when I think of what that money could be put to use for, I become less sure of my preference for it. 

Just to be clear, is this for the company name, the app, or both?

**Comment by skddc** [comment]
Both.

This money is only being offered for the domain, because we think it's a great name and address. It wouldn't be put to use for something else at this point in time (or later for that matter).

**Comment by skddc** [comment]
@silverbucket Any other questions?

**Comment by bumi** [comment]
I do like the name. seems easy to remember. matches the theme. :+1: 
was just wondering if it is worth spending the money. also because most of the country specific TLDs are taken.

**Comment by skddc** [comment]
Why would we need country TLDs?

**Comment by skddc** [comment]
FYI: I edited @silverbucket's comment. I'd prefer not to put financial information in public repos yet.

**Comment by skddc** [comment]
@bumi Could you elaborate on the country TLD point?

**Comment by bumi** [comment]
sorry, what I've actually meant is not those new TDLs (like .email) but the general .com domain for example or country specific domains like .de

**Comment by skddc** [comment]
Yes, that's what my question is about. Why would we need a `.de` e.g.?

**Comment by skddc** [comment]
The `.com` is available but quite expensive. Obviously any such name will be very expensive as a `.com`, if not being used already.

I don't think we necessarily need it as single word, though. Would only be for pro, so we could either use sth like `pro.kosmos.org` or something with a combined word as a `.com`, or something else entirely.

I believe the popularity of it as an open source project will define commercial success, so my personal first goal and proposed common goal is to focus on an awesome start as open source project first and think about Pro only if the MVP actually kicks ass and people like it enough to want to pay for it for their teams.

**Comment by skddc** [comment]
By the way, if anyone has more names to propose, please do so!!!

**Comment by skddc** [comment]
@silverbucket @bumi Please keep commenting, so we can make meaningful progress!

@bkero @dkgrieshammer Still waiting for you guys. ;)

**Comment by dkgrieshammer** [comment]
I think Kosmos is quite cool - one thing we should think about is the possibility to transform the name to an action verb if you want users to memorize your app - I'll skype you later, I'll slack you, I'm tweeting, I whats-app you (whatsapp is not really working)
- so you would maybe 'cos somebody? Or Messages are Kosmos Beams so I beam you up?
I think we can work with kosmos

**Comment by skddc** [comment]
> I'll skype you later, I'll slack you, I'm tweeting, I whats-app you (whatsapp is not really working)

I think Slack also isn't working there.

> so you would maybe 'cos somebody? Or Messages are Kosmos Beams so I beam you up? I think we can work with kosmos

Haha, cos me, bro! I'll beam you up later.

I love beaming! Cool idea.

**Comment by skddc** [comment]
@bumi @silverbucket @bkero Please keep the comments coming until we have reached a conclusion!

**Comment by skddc** [comment]
Current status is 5 out of 8 like it enough to be +1 by my count. Would be great to either get the other 3 on board or decide against it and have new proposals to decide on asap.

**Comment by bkero** [comment]
+1 for Kosmos from me

**Comment by skddc** [comment]
@silverbucket, @bumi?

**Comment by silverbucket** [comment]
Sorry,  been travelling.

Things that first came to mind with the word kosmos: Russian kosmonauts,
Cosmo women's fashion magazine.

The extra s in the name could easily be dropped by people in which case
we'd be audibly the same as the women's magazine.

I gave it some time to see if it would grow on me but it really hasn't.
That said, I'm OK with using it if everyone else likes it. I still like
`67p` as a company name with the launch of the basic product as a
subdomain, which gives us more flexibility it choosing a name. But
understand we can't be held up in a name discussion forever.

I also agree about it being a verb. Slack is used like that, though not as
ubiquitous as Skype or Facebook. Though this tend to find their way into
the lingo once they reach a certain critical mass.

**Comment by skddc** [comment]
> The extra s in the name could easily be dropped by people in which case we'd be audibly the same as the women's magazine.

1. How can you drop the S easily? You wouldn't drop it from other words like e.g. "Cileos" (one of the companies in our office).
2. There is no such women's magazine. I think you're referring to [Cosmopolitan](http://www.cosmopolitan.com/). Good thing nobody else had that in mind so far. ;)

> I gave it some time to see if it would grow on me but it really hasn't.

That's unfortunate. :( I had hoped for everybody liking it.

> I also agree about it being a verb. Slack is used like that, though not as ubiquitous as Skype or Facebook. Though this tend to find their way into the lingo once they reach a certain critical mass.

I never saw or heard Slack being used (also, that's already a different word), as is the case with Facebook. Maybe it's different between native English speakers? Anyway, as I said before, I usually hate using product names as verbs and I think most of us do, but @dkgrieshammer's ideas were pretty awesome. I think even *better* than using the product name, both in concept and execution.

Anyway, too bad it's not a favorite of yours! You can still throw new names in the mix, of course!

Waiting is indeed not an option. We can't tell everybody that we want to change the world, but we're not even able to come up with a name for our software to begin with. I think that would make us appear uncertain and slightly incompetent.

**Comment by bumi** [comment]
The thought about the other TDLs came from the idea to own most of the domains to prevent a naming or SEO conflict with other companies. But I guess that is not really important for a product like _kosmos_. - but more for a "end user service" like AirBnB. 

I like kosmos and the theme that can be built around the name. (also @dkgrieshammer's idea)

**Comment by skddc** [comment]
![its-happening](https://cloud.githubusercontent.com/assets/842/6658929/2ce848f8-cb70-11e4-9181-aba00c3f72c4.gif)

**Comment by silverbucket** [comment]
> How can you drop the S easily? You wouldn't drop it from other words ending in S like e.g. "Cileos" (one of the companies in our office).

Laziness? misunderstanding? Not saying it *will* happen, just that it's so similar to "Cosmo" that people could slip up or mishear quite easily.

> There is no such women's magazine. I think you're referring to Cosmopolitan. Good thing nobody else had that in mind so far. ;)

Everyone calls it Cosmo.

> Waiting is indeed not an option. We can't tell everybody that we want to change the world, but we're not even able to come up with a name for our software to begin with. I think that would make us appear uncertain and slightly incompetent.

Well. We never needed a name to get started, the plan was to have basic functionality completed during hackerbeach, name or not. And just the same, getting a name now doesn't mean we can finally begin development, does it? Also, we're merging naming the product and the company into one thing and I think that makes it more difficult.

FYI: I'm only online occasionally this week as I'm visiting my Grandma and her internet connection is horrid.

**Comment by skddc** [comment]
You're mixing some things up there. Nobody said anything about naming the company or even founding a company yet. And nobody said software development is held up by the naming.

> Everyone calls it Cosmo.

Well, as you see, demonstrably not everyone.

**Comment by skddc** [comment]
By the way, I got the price for the domain down some more, but we're having issues with our PayPal account.

**Comment by gregkare** [comment]
I only had heard the magazine being called Cosmo in Sir Mix-A-Lot's Baby Got Back, I didn't know it was common in the US

**Comment by skddc** [comment]
Too bad there's no way to fav/like comments on GitHub. :)

**Comment by silverbucket** [comment]
On Mon, Mar 16, 2015, 17:17 Sebastian Kippe <notifications@github.com>
wrote:

You're mixing some things up there. Nobody said anything about naming the
company or even founding a company yet. And nobody said software
development is held up by the naming.

 Oh,  I asked earlier if this was for a company name as well as app name
and thought you said yes.

 Everyone calls it Cosmo.


Well, as you see, demonstrably not everyone.

 People in America, clearly I'm not talking about outside America. Cosmo is
very well established here. I just brought up the magazine to my sister and
she said she calls it cosmos for fun (haven't mentioned anything about
kosmos), which I thought was funny. Just giving you an American
perspective.

—
Reply to this email directly or view it on GitHub
<https://github.com/67P/meta/issues/1#issuecomment-82002444>.<img src="
https://ci4.googleusercontent.com/proxy/OJ4K9Ei3dIoUyEsJdZw-je_E2zN7BCMKHpSdrQEXVwVc2hJs43TBnJ1gdrARNrH7ETqi3Y7zIG1DgeAaZV5kPlzfhFehk_IveKiZr1bBRAzJsVd5Oz583T7deSgyNWFxySYlOa44wi21NmPvg_PwISSpJGjJAg=s0-d-e1-ft#https://github.com/notifications/beacon/AATYg8OdK3xia_hlUYkkABj7DTPEI6VLks5n12oTgaJpZM4DuShv.gif
">
